py3status/modules/sysdata.py

```diff
@@ -349,11 +352,8 @@ def sysdata(self):
 
         # get load average
         if self.py3.format_contains(self.format, 'load*'):
-            load1, load5, load15 = self._get_loadavg()
-            sys['load1'] = load1
-            sys['load5'] = load5
-            sys['load15'] = load15
-            self.py3.threshold_get_color(load1, 'load')
+            sys.update(dict(zip(self.keys['load'], self._get_loadavg())))
+            self.py3.threshold_get_color(sys['load1'], 'load')
```

Comment:
nice but maybe like this?

```python
load_data = self._get_loadavg()
sys.update(zip(['load1', 'load5', 'load15'], load_data))
self.py3.threshold_get_color(load_data[0], 'load')
```

Reply:
Oh no `dict`? Sure. I was going to do that later... Ha. Look at the other PR. I also removed `/proc/loadavg` in favor of `from os import getloadavg`.... It kills few lines... do you want me to do that here too? (eg, just `get_loadavg()))...`) instead of `load_data`? About to head out for an hour soon.

Reply:
P.S. There will be lot of keys... so do you want to do that for all of them too?

Reply:
for me it is more readable if the keys are where they are used - I think they are just used in one place.

> do you want me to do that here too? 

small PRs == quick merges

> /proc/loadavg in favor of from os import getloadavg

it'd be nice to see the difference, maybe do several thousand runs under time and see if it makes a difference

Reply:
Zipped.